client/app/models/affected-zoning-resolution.js

```diff
@@ -12,4 +12,7 @@ export default class ProjectActionModel extends Model {
 
```

Comment:
Should the class name here be `AffectedZoningResolutionModel`, instead of `ProjectActionModel`?

Reply:
Oh whoops yes it should -- looks like that was left over from previous stuff